src/main/java/com/conveyal/datatools/manager/jobs/MergeFeedsJob.java

```diff
@@ -0,0 +1,584 @@
+        ByteArrayOutputStream tableOut = new ByteArrayOutputStream();
+        Map<String, String[]> rowValuesForKeys = new HashMap<>();
+        Set<String> rowStrings = new HashSet<>();
+        int mergedLineNumber = 0;
+        // Get the spec fields to export
+        List<Field> specFields = table.specFields();
+        boolean stopCodeMissingFromFirstTable = false;
+        try {
+            // Iterate over each zip file.
+            for (int feedIndex = 0; feedIndex < feedsToMerge.size(); feedIndex++) {
+                mergeFeedsResult.feedCount++;
+                // FIXME add check for merge type.
+                if (feedIndex > 0 && (table.name.equals("agency") || table.name.equals("feed_info")) && mergeType.equals(MTC)) {
+                    // Always prefer future file for agency and feed_info tables, which means that we can skip
+                    // iterations following the first one.
+                    // FIXME: This could cause issues with routes or fares that reference an agency_id that no longer
+                    //  exists.
+                    LOG.warn("Skipping {} file for feed {}/{} (future file preferred)", table.name, feedIndex, feedsToMerge.size());
+                    continue;
+                }
+                FeedToMerge feed = feedsToMerge.get(feedIndex);
+                FeedVersion version = feed.version;
+                FeedSource feedSource = version.parentFeedSource();
+                // Generate ID prefix to scope GTFS identifiers to avoid conflicts.
+                String idScope = getCleanName(feedSource.name) + version.version;
+                CsvReader csvReader = table.getCsvReader(feed.zipFile, null);
+                // If csv reader is null, the table was not found in the zip file. There is no need to handle merging
+                // this table for the current zip file.
+                if (csvReader == null) {
+                    LOG.warn("Table {} not found in the zip file for {}{}", table.name, feedSource.name, version.version);
+                    continue;
+                }
+                LOG.info("Adding {} table for {}{}", table.name, feedSource.name, version.version);
+
+                Field[] fieldsFoundInZip = table.getFieldsFromFieldHeaders(csvReader.getHeaders(), null);
+                List<Field> fieldsFoundList = Arrays.asList(fieldsFoundInZip);
+                // Determine the index of the key field for this version's table.
+                int keyFieldIndex = getFieldIndex(fieldsFoundInZip, keyField);
+                int lineNumber = 0;
+                // Iterate over rows in table, writing them to the out file.
+                while (csvReader.readRecord()) {
+                    String keyValue = csvReader.get(keyFieldIndex);
+                    // FIXME add check for merge type?
+                    if (mergeType.equals(MTC) && lineNumber == 0 && table.name.equals("stops")) {
```

Comment:
There's already a check for the mergeType here. Can you elaborate on or remove the above comment?